tests/dataset_test.py

```diff
@@ -936,19 +936,34 @@ def test_is_edges():
 def test_drop_coords():
     coord0 = sc.linspace('x', start=0.2, stop=1.61, num=4)
     coord1 = sc.linspace('y', start=1, stop=4, num=3)
-    data0 = sc.array(dims=['x', 'y'], values=np.random.rand(4, 3))
+    coord2 = sc.linspace('z', start=-12, stop=0, num=5)
+    data0 = sc.array(dims=['x', 'y', 'z'], values=np.random.rand(4, 3, 5))
     data1 = sc.array(dims=['x'], values=np.random.rand(4))
     da = sc.Dataset(data={
         'data0': data0,
         'data1': data1
     },
                     coords={
                         'coord0': coord0,
-                        'coord1': coord1
+                        'coord1': coord1,
+                        'coord2': coord2
                     })
+
+    assert 'coord0' not in da.drop_coords('coord0').coords
+    assert 'coord1' in da.drop_coords('coord0').coords
+    assert 'coord2' in da.drop_coords('coord0').coords
+    assert 'coord0' not in da.drop_coords('coord0')['data0'].coords
+    assert 'coord0' not in da.drop_coords('coord0')['data1'].coords
+    assert 'coord1' in da.drop_coords('coord0')['data0'].coords
```

Comment:
I think these are not necessary, since this is just re-testing the behavior of `Dataset.__getitem__`, which is tested elsewhere? OR did you have a particular reason for including these assertions?

Reply:
Not really...? I also thought it might not be necessary. I just removed it